src/prefect/utilities/dockerutils.py

```diff
@@ -428,7 +428,7 @@ def push_image(
     """
 
     if not tag:
-        tag = slugify(now("UTC").isoformat())
+        tag = slugify(prefect.types._datetime.now("UTC").isoformat())
```

Comment:
Some places use the full path for `now` to allow mocking the current time for tests.